src/intl/resolver.js

```diff
@@ -329,9 +325,9 @@ function Pattern(env, ptn) {
             `(${str.length}, max allowed is ${MAX_PLACEABLE_LENGTH})`
           )
         );
-        result += FSI + str.substr(0, MAX_PLACEABLE_LENGTH) + PDI;
+        result += str.substr(0, MAX_PLACEABLE_LENGTH);
```

Comment:
since this is a hot path, can you test array and `.join` at the end instead of concatenation?